doc/nrf/ug_bt_mesh.rst

```diff
@@ -0,0 +1,91 @@
+.. _ug_bt_mesh:
+
+Working with Bluetooth Mesh
+###########################
+
+The |NCS| provides support for developing applications using the Bluetooth Mesh protocol.
+The support is based on Zephyr's :ref:`bluetooth_mesh` implementation.
+
+.. _bt_mesh_ug_intro:
+
+Introduction
+************
+
+The `Bluetooth Mesh Profile Specification`_ is developed and published by the Bluetooth SIG.
+It allows one-to-one, one-to-many, and many-to-many communication, using
```

Comment:
```suggestion
It allows one-to-one, one-to-many, and many-to-many communication, using the Bluetooth Low Energy protocol to exchange messages between the nodes on the network.
```
Merge line 15 and 16.

Reply:
We can now also use |BLE| for all BLE occurrences.